scripts/package-tests/README.md

```diff
@@ -0,0 +1,26 @@
+# Build and test deb/rpm/apk packages
+
+## Prerequisites
+
+Tools:
+
+- [Go](https://go.dev/)
+- [GoReleaser](https://goreleaser.com/)
+- [Podman](https://podman.io/)
+- make
+
+## How to build and test
+
+To build the collector linux packages, a few steps are required:
+
+- Run `make generate` to (re-)generate sources and goreleaser files
+- Go to the distribution folder that you want to build (under the `distributions` folder)
+- Run `goreleaser release --snapshot --clean --skip sbom,sign,archive,docker`
+    - This will build a full release with all architectures and packaging types into the `dist` folder inside your
+      current folder. (We can skip many parts of the release build that we don't need)
```

Comment:
I don't understand the comment about skipping parts. Who is 'we'? How can these parts be skipped? Why are they not needed?

Reply:
we as in the users :)
many of the goreleaser steps can be skipped for the package tests as they are not needed for those.

Reply:
Thanks for clarifying! So this comment is also intended for developers running the tests locally then?

Reply:
I updated the wording slightly to hopefully make it a bit clearer